### PR TITLE
makes a bunch of job-given mutations magical

### PR DESCRIPTION
**Pull request by zjdtmkhzt**
## About the PR 
This makes a bunch of the mutations you might get because of your job magical.
This means they will not cost stability and will not be curable/researchable via mutadone/genetics.
## Why's this needed? 
Mainly because the Halloween jobs had less than 100 stability because of this, making it so that angels occasionally did a whole bunch of damage to themselves with their healing powers and stuff like that.

Balance is somewhat debatable due to mutations being hard to remove with this, but I think it'd be fine, since most of the mutations aren't that dangerous.
(except blue clowns having regen for some reason. :o)

If this is deemed too strong, they could just be made to not cost stability instead.

This also makes it so that mimes can no longer cure their mutism with mutadone. (Damn casuals.)

### CODE REVIEW

**Comment by zagoon** [comment]
I dislike preventing miners and chefs from curing their mutation when it's randomly assigned; it can be particularly bad for chefs if they already had an accent trait.

**Comment by zjdtmkhzt** [comment]
Yeah, I can see that!
Don't see the issue with miners, but I assume mutantrace can probably be changed anyway with a genetics machine.

File: code/datums/jobs.dm [hunk 1]
```diff
@@ -708,7 +708,7 @@
 			return
 		M.bioHolder.AddEffect("training_miner")
 		if (prob(20) && !M.mutantrace)
-			M.bioHolder.AddEffect("dwarf") // heh
+			M.bioHolder.AddEffect("dwarf", magical=1) // heh
```
Comment (ZeWaka) [inline code/datums/jobs.dm:711]: heh

Comment (zjdtmkhzt) [reply code/datums/jobs.dm:711]: heh

**Comment by DabirSA** [comment]
> 
> 
> I dislike preventing miners and chefs from curing their mutation when it's randomly assigned; it can be particularly bad for chefs if they already had an accent trait.

You could have job-assigned accents override other accent traits, maybe?

**Comment by Tiggersaurus** [comment]
Appreciate this. Using the angel heal and then suddenly being in crit was a shock lol.

**Comment by zjdtmkhzt** [comment]
Having job accents override trait accents would work, but I think people would rather have it the other way around.
That can always be handled in a later patch though, if desired, it feels a bit out of scope for this.
Not sure how often the situation comes up, I feel like I rarely see people use the accent traits, and many of those players revel in being unintelligible.^^

**Pull request merged**